net-libs/tox/tox-9999.ebuild

```diff
@@ -97,15 +101,12 @@ pkg_postinst() {
 		ewarn "causes an infinite loop for certain rev-dep-rebuild"
 		ewarn "commands. If you aren't running a node, please"
 		ewarn "consider disabling the dht-node flag"
-	fi
-	if use daemon; then
-		if [[ -f ${EROOT}/var/lib/tox-dht-bootstrap/key ]]; then
-			ewarn "Backwards compatability with the bootstrap daemon might have been"
-			ewarn "broken a while ago. To resolve this issue, REMOVE the following files:"
-			ewarn "    ${EROOT}/var/lib/tox-dht-bootstrap/key"
-			ewarn "    ${EROOT}/etc/tox-bootstrapd.conf"
-			ewarn "    ${EROOT}/run/tox-dht-bootstrap/tox-dht-bootstrap.pid"
-			ewarn "Then just re-emerge net-libs/tox"
-		fi
+		ewarn ""
```

Comment:
Have these issues been reported upstream? Could you provide a bug reference?

Reply:
Yes, I will include a bug reference.